src/tests/ftest/util/file_count_test_base.py

```diff
@@ -49,21 +53,27 @@ def run_file_count(self):
         ior_ppn = self.params.get("ppn", '/run/ior/client_processes/*', None)
         mdtest_np = self.params.get("np", '/run/mdtest/client_processes/*', 1)
         mdtest_ppn = self.params.get("ppn", '/run/mdtest/client_processes/*', None)
+        intercept = os.path.join(self.prefix, 'lib64', 'libpil4dfs.so')
+
+        ior_oclass = object_class[0]
+        mdtest_oclass = object_class[1]
+        
         # create pool
         self.add_pool(connect=False)
 
-        for oclass in object_class:
-            self.ior_cmd.dfs_oclass.update(oclass)
-            self.mdtest_cmd.dfs_oclass.update(oclass)
-            self.ior_cmd.dfs_dir_oclass.update(oclass)
+        for idx, _ in enumerate(object_class):
+            self.ior_cmd.dfs_oclass.update(ior_oclass[idx])
+            self.mdtest_cmd.dfs_oclass.update(mdtest_oclass[idx])
+            self.ior_cmd.dfs_dir_oclass.update(ior_oclass[idx])
```

Comment:
This is enumerating `object_class` but then using the index like `ior_oclass[idx]` which I think is confusing. What if `object_class` is longer than `ior_oclass`?